addons/survey/report/survey_templates.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="certification_report_view_general">
+            <!-- Style classes to be applyed to '#o_survey_certification': [no class](purple), gold, blue -->
+            <div id="o_survey_certification" t-att-data-oe-model="user_input._name" t-att-data-oe-id="user_input.id" 
+                t-attf-class="#{'article certification-wrapper ' + layout_template + ' ' + layout_color}">
+                <div class="certification">
+                    <div class="certification-seal" t-if="user_input.scoring_success and layout_template == 'modern'"/>
+                    <div class="certification-top">
+                        <h1><b>Certificate</b>
+                            <br/><span t-if="user_input.scoring_success">of achievement</span>
+                        </h1>
+                    </div>
+
+                    <div class="page certification-content">
+                        <div class="oe_structure"/>
+                        <div t-if="user_input.scoring_success">
+                        <div class="oe_structure"/>
+                            <p> <span>This certificate is presented to</span>
+                                <br/>
+                                <t t-set="certif_style" t-value="''"/>
+                                <t t-set="certified_name" t-value="user_input.partner_id.name or user_input.email or ''"/>
+                                <t t-if="certified_name.isupper()">
+                                    <t t-set="certif_style" t-value="certif_style + 'font-family: certification-serif;'"/>
+                                </t>
+                                <t t-if="len(certified_name) > 20 and layout_template == 'modern'">
+                                    <t t-set="certif_style" t-value="certif_style + 'font-size: 40px; line-height: 4;'"/>
+                                </t>
```

Comment:
You're right, I did not read this part properly.

Maybe write it this way to make it a bit more readable?

```
if > 35 and classic:
    ...
elif > 20:
    if modern
        ....
    else:
        ....
```

?

Currently it's a bit hard to follow.